### ISSUE TITLE
API seems to have changed to v1/tts

### ISSUE THREAD

**Issue by NinjaPerson24119** [Ph0rk0z/fish-speech-silly#2]
### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/fishaudio/fish-speech/discussions).
- [X] I have searched for existing issues [search for existing issues](https://github.com/fishaudio/fish-speech/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/fishaudio/fish-speech/issues/515)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Cloud or Self Hosted

Self Hosted (Source)

### Steps to reproduce

`${this.settings.provider_endpoint}/v1/invoke` should be v1/tts

**Comment by NinjaPerson24119** [comment]
Love the code btw, integrates nicely. Thanks for making this.

**Comment by Ph0rk0z** [comment]
Yes, it's changed to support the way I had it in sillytavern. I think main repo has a non packed API now but still doesn't support voices on the server.
